tox.ini

```diff
@@ -17,10 +17,17 @@ setenv = VIRTUAL_ENV={envdir}
          TESTS_DIR={[testenv]unit_tests}
          PYFILES={toxinidir}/setup.py {toxinidir}/hotsos/ {[testenv]unit_tests}
          non-utc-tz: TZ=EST+5
+         PYTHON=coverage run --source hotsos --parallel-mode
 deps =
     -r{toxinidir}/requirements.txt
     -r{toxinidir}/test-requirements.txt
-commands = stestr run --serial --test-path {[testenv]unit_tests} {posargs}
+commands =
+    coverage erase
+    stestr run --serial --test-path {[testenv]unit_tests} {posargs}
+    coverage combine
+    coverage report
```

Comment:
Maybe we can add `--fail-under=30`? The lowest I currently see is 32%, so it won't fail for existing code.
At the same, we can enforce a minimum level is covered in the future?

Reply:
I would think we'd want to set a minimum so we don't regress if possible if this will be part of our gating. The other thing done in some of the openstack projects is to use --skip-covered to not report on files with 100% coverage.

coverage report --fail-under=XX --skip-covered